### ISSUE TITLE
can't find .SimulateTextEntry

### ISSUE THREAD

**Issue by xychix** [michaelnoonan/inputsimulator#24]
InputSimulator.SimulateTextEntry("HELLO!");

this function is mentioned in the readme but I can't seem to find it in the source at all.
Am I missing some parts? (I'm not a dev-ver)

**Comment by TThanushan** [comment]
It seems that the method name is **TextEntry()**.

This should work : 
```
InputSimulator sim = new InputSimulator();`
sim.Keyboard.TextEntry("HELLO!");
```

**Comment by dposea** [comment]
SimulateKeyPress is also not found. I spent way too much time figuring out it  did not exist.

**Comment by DavidInvenio** [comment]
> SimulateKeyPress is also not found. I spent way too much time figuring out it did not exist.

Yeah me too.  Even downloaded and compiled the fork and it's the same. I kept thinking I had gotten an old build of it somehow.
It's not that I don't appreciate the library, open and free to use.  
But how hard can it be to update a single page of example code?

**Comment by codeulike** [comment]
oh crap. So its more like a DontReadMe.md
